apps/dashboard/app/(app)/logs/components/filters/components/response-status.tsx

```diff
@@ -0,0 +1,92 @@
+import { Checkbox } from "@/components/ui/checkbox";
+import { Popover, PopoverContent, PopoverTrigger } from "@/components/ui/popover";
+import React, { useEffect, useState } from "react";
+import { type ResponseStatus as Status, useLogSearchParams } from "../../../query-state";
+
+interface CheckboxItemProps {
+  id: string;
+  label: string;
+  description: string;
+  checked: boolean;
+  onCheckedChange: (checked: boolean) => void;
+}
+
+const checkboxItems = [
+  { id: "500", label: "Error", description: "500 error codes" },
```

Comment:
let's go with "5xx error codes" etc, to indicate that it's not just 500, but also 503 etc